SS_write_report.tpl

```diff
@@ -1305,7 +1305,7 @@ FUNCTION void write_bigoutput()
   {
     SS2out << endl
            << pick_report_name(15) << endl;
-    SS2out << "Fleet Fleet_Name Area Yr Seas Time Obs Exp Mult Exp*Mult se F  Like sel_bio kill_bio ret_bio sel_num kill_num ret_num" << endl;
+    SS2out << "Fleet Fleet_Name Area Yr Seas Time Obs Exp Mult Exp*Mult se F  Like sel_bio kill_bio ret_bio sel_num kill_num ret_num vuln_bio_mid vuln_num_mid" << endl;
```

Comment:
Do we care that there is now a mix of biomass and numbers rather than before all biomass columns came before the numbers columns, either can be accommodated in r4ss, I am just trying to think of others? Maybe it is best to tack them on the for those that use numerical reference to columns but for aesthetic and future code it seems more logical to keep them grouped. Second, I would have named the new columns `vuln_mid_bio` and `vuln_mid_num` to retain the _bio and _num at the end of the column name rather than placing it in the middle.

Reply:
kill and ret don't apply to the vuln calculation.
all other columns use "mid" also, so perhaps just drop that qualifier?
so could be:  vuln_bio sel_bio kill_bio ret_bio vuln_num .... 
then perhaps but the defining eqs above or below the table?